src/components/OSCALProfileCatalogInheritance.js

```diff
@@ -21,26 +21,31 @@ function renderTree(nodes, id) {
   const generateLabel = (title, type) =>
     type === "profile" ? `Profile: ${title}` : `Catalog: ${title}`;
 
+  const idReassign = id;
   const children = [];
 
   nodes.inherited.forEach((node) => {
-    id[0] += 1;
+    idReassign[0] += 1;
```

Comment:
We're not really short on characters here. How about `inheritedNodeId` or just `nodeId` to give it a more descriptive name?

Reply:
Another thought here would be to add the UUID of the OSCAL document to the inheritance object, but we also talk about the possibility of using default icons at the `TreeView` level, so this ID may not be needed at all.